permissions/mod.ts

```diff
@@ -44,7 +45,10 @@ function getPermissionString(descriptors: Deno.PermissionDescriptor[]): string {
  * ```
  *
  * If one of the permissions requires a prompt, the function will attempt to
- * prompt for it.  The function resolves with all of the granted permissions. */
+ * prompt for it.  The function resolves with all of the granted permissions.
+ *
+ * @deprecated (will be removed in 1.0.0) Use the Deno permission API instead
```

Comment:
Is the deprecation notice present when using every overload or just this one? If just the one, I'd have a sweep through the codebase and other current PRs to make the appropriate adjustments.

Reply:
In this specific case, it's fine since we've deprecated ever individual overload (outside of the base one, which seems inaccessible? Could I get you to check that?), but I'd definitely make sure we've properly deprecated every overload.

Speaking of checking the inaccessibility of the base thing. These two work:
```ts
grant({name: "net"}, {name: "net"})
grant([{name: "net"}, {name: "net"}])
```

This doesn't:
```ts
grant([{name: "net"}], {name: "net"})
```

Why? Am I reading this type wrong?

Reply:
Ah, whoops didn't see that each overload had the declaration. Either way, I'd check elsewhere if an overload is missing the declaration.